src/test/java/MySweetProgram.java

```diff
@@ -16,4 +17,42 @@ public void titleTest() {
         driver.quit();
     }
 
+    @Test
+    public void LoginWrongCredentialTest() throws InterruptedException {
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://testkwidos.tk/auth/login");
+        driver.manage().window().maximize();
```

Comment:
Please move driver configuration to another method, because you use this configuration in several test methods and also test should follow this AAA pattern: https://cypriops.com/blog/test-food-chain/